include/solvers/multiphysics_interface.h

```diff
@@ -840,6 +938,14 @@ class MultiphysicsInterface
 
   std::map<PhysicsID, DoFHandler<dim> *> physics_dof_handler;
 
+
+  std::map<PhysicsID, TrilinosWrappers::SparseMatrix *> physics_matrix;
+  std::map<PhysicsID, TrilinosWrappers::MPI::Vector *>  physics_rhs;
```

Comment:
Very unclear to me why these would need to be passed around...

Reply:
This was used in another architecture but it's not used anymore.